prowler/providers/aws/aws_provider.py

```diff
@@ -650,14 +652,15 @@ def generate_regional_clients(
                 f"{error.__class__.__name__}[{error.__traceback__.tb_lineno}]: {error}"
             )
 
-    def get_available_aws_service_regions(self, service: str) -> set:
+    @staticmethod
+    def get_available_aws_service_regions(
+        service: str, partition: str, audited_regions: set = None
```

Comment:
`partition` should be also optional.

Reply:
Could you add a docstring to this function? thanks!

Reply:
Done